packages/babel-helper-compilation-targets/src/index.js

```diff
@@ -179,6 +179,7 @@ export default function getTargets(
   options: GetTargetsOption = {},
 ): Targets {
   let { browsers, esmodules } = inputTargets;
+  const { configPath = "." } = options;
```

Comment:
Unlike browserslist, we use `"."` instead of `path.resolve(".")` here to avoid `import "path"` so we don't have to add a `packageJson.browsers` field for this package.

`browserslist` will use `path.resolve` when it is running in a Node environment: 

https://github.com/browserslist/browserslist/blob/209adf9e0051fa39a2b25354cffd493300f34b02/node.js#L302